tests/migrations/test_commands.py

```diff
@@ -695,6 +695,40 @@ def test_sqlmigrate_for_non_transactional_databases(self):
             self.assertNotIn(start_transaction_sql.lower(), queries)
         self.assertNotIn(connection.ops.end_transaction_sql().lower(), queries)
 
+    @override_settings(MIGRATION_MODULES={"migrations": "migrations.test_migrations_squashed"})
+    def test_sqlmigrate_ambigious_squashed_migration_name(self):
```

Comment:
Two tests i.e. `test_sqlmigrate_ambigious_squashed_migration_name()` and `test_sqlmigrate_squashed_migration()` work without this patch. It's fine to increase coverage but we should move them to a separate commit.